bokeh/embed/bundle.py

```diff
@@ -420,7 +420,13 @@ def _use_mathjax(objs: Sequence[Model | Document]) -> bool:
         bool
     '''
     from ..models.text import MathText
-    return _any(objs, lambda obj: isinstance(obj, MathText))
+
+    def model_require_mathjax(model: Model) -> bool:
+        ''' If any model explicit require MathJax with enable_tex option set to true
+        '''
+        return "enable_tex" in model.properties() and model.enable_tex
```

Comment:
This will also need to be impoved in the future, it can go along with the same other issue I mentioned creating in the other PR. For the immediate term I do think this would be preferred than mucking with string property names:
```
return isinstance(model, Markup) and model.enable_tex
```